assets/headers/blast_outfmt6_header.txt

```diff
@@ -0,0 +1 @@
+stitle	qaccver	saccver	pident	length	mismatch	gapopen	qstart	qend	sstart	send	evalue	bitscore	slen	qlen	qcovs	%cgAligned	%refCovered
```

Comment:
What is the purpose of this file?

Reply:
It is a file defining the header for BLAST output. Easier to change this in one place in a file then to define it in each process:
https://github.com/nf-core/viralrecon/blob/5ad5c00c81bb67c3eb32d87cf21c85b7d3c7e866/main.nf#L1935